CHANGELOG.md

```diff
@@ -21,6 +75,8 @@ We'd like to offer a big thanks to the 15 contributors who made this release pos
 
 ### `@mui/x-data-grid@v5.12.0` / `@mui/x-data-grid-pro@v5.12.0` / `@mui/x-data-grid-premium@v5.12.0`
 
+#### Changes
```

Comment:
This was missed last time. Its just for consistency

Reply:
yes, but I checked the older releases and it seems we are always adding it. It does seem more structured with it.